services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/IntentionalRerouteSpec.groovy

```diff
@@ -283,6 +284,62 @@ class IntentionalRerouteSpec extends HealthCheckSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    @Tags(HARDWARE)
+    def "Intentional flow reroute with VXLAN encapsulation is not causing any packet loss"() {
+        given: "A vxlan flow"
+        def allNoviflowSwitchIds = topology.activeSwitches.findAll { it.noviflow }*.dpId
+        def allTraffgenSwitchIds = topology.activeTraffGens*.switchConnected.findAll {
+            allNoviflowSwitchIds.contains(it.dpId)
+        }*.dpId ?: assumeTrue("Should be at least two active traffgens connected to NoviFlow switches",
+                false)
+        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
+            it.src.noviflow && it.dst.noviflow &&
+            allTraffgenSwitchIds.contains(it.src.dpId) && allTraffgenSwitchIds.contains(it.dst.dpId)
```

Comment:
`it.src.noviflow && it.dst.noviflow` conditions are excessive, since `allTraffgenSwitchIds` is guaranteed to contain only noviflow switches